app/form.tsx

```diff
@@ -31,21 +32,9 @@ export function PollCreateForm() {
         }
     });
 
-    const pollStub = {
-        id: uuid(),
-        created_at: new Date().getTime(),
-        title: '',
-        option1: '',
-        option2: '',
-        option3: '',
-        option4: '',
-        votes1: 0,
-        votes2: 0,
-        votes3: 0,
-        votes4: 0,
-        validInDays: MIN_VALID_IN_DAYS,
-    };
-    const saveWithNewPoll = savePoll.bind(null, pollStub);
+    const pollStub = createNullPoll();
+    pollStub.id = uuid();
+    pollStub.created_at = Date.now();
```

Comment:
Assign id and created_at in `createNullPoll`.

```suggestion
    const pollStub = createNullPoll();
    pollStub.id = uuid();
    pollStub.created_at = Date.now();
```